common/src/main/java/org/wso2/testgrid/common/DashboardSetup.java

```diff
@@ -60,7 +68,7 @@ public DashboardSetup(String testplanID) {
     public void initDashboard() {
         // create influxDB database according to tp_id
         try {
-            InfluxDB influxDB = InfluxDBFactory.connect(restUrl + ":8086", username, password);
+            InfluxDB influxDB = InfluxDBFactory.connect("http://" + restUrl + ":8086", username, password);
```

Comment:
`"http://"`  string is used in several places. So better we declare it as a Constant.